compute_tools/src/http/api.rs

```diff
@@ -51,6 +52,20 @@ async fn routes(req: Request<Body>, compute: &Arc<ComputeNode>) -> Response<Body
             Response::new(Body::from(serde_json::to_string(&status_response).unwrap()))
         }
 
+        // prometheus metrics
+        (&Method::GET, "/metrics") => {
```

Comment:
do you expect to scrape these metrics by Prometheus?

Reply:
yes, that's the plan